BackBLAST

```diff
@@ -8,12 +8,10 @@ set -euo pipefail
 readonly VERSION="0.8.0"
 readonly SCRIPT_NAME="${0##*/}"
 readonly SCRIPT_DIR="$(realpath ${0%/*})"
+readonly UTILS_DIR="${SCRIPT_DIR}/scripts"
```

Comment:
Here, I define the path to the directory where utility scripts (e.g., `search.py`) are found. This is different than the old way the code used to work, where all of the utility scripts were fully exposed on the user's PATH.

Reply:
@jmtsuji Looks fine.